mewarpx/requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy>=1.21.0
+psutil>=5.8.0
+scipy>=1.7.0
```

Comment:
Aren't there a few more dependencies? From looking at the slab doc you also have in the list:
- matplotlib v3.2.2 (must be this version)
- openpmd
- yt
- mpi4py

Reply:
These are just the dependencies for mewarpx. We're not actually installing pywarpx for the autodoc, it's just being mocked for the mewarpx installation, since we're not documenting the pywarpx modules.

Reply:
@KZhu-ME Actually have you retried without this file? Normally I would hope any sort of install / pip-based install would be able to read setup.py 's requirements format. I mean you just added pandas there and not here for example...

Reply:
It should work without this file. I'll test it out without it, and remove it if it's unneeded.